bearer scan on main causes panic

<!--- Provide a general summary of the issue in the Title above -->

## Description & Reproduction

`bearer scan` causes a panic

## Expected Behavior

returns help message

## Actual Behaviour
```
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/bearer/bearer/pkg/commands.NewScanCommand.func2(0x14001240900, {0x104789ec0?, 0x0, 0x0})
	/Users/runner/work/bearer/bearer/pkg/commands/scan.go:58 +0x378
github.com/spf13/cobra.(*Command).execute(0x14001240900, {0x104789ec0, 0x0, 0x0})
	/Users/runner/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:916 +0x5c8
github.com/spf13/cobra.(*Command).ExecuteC(0x14001240000)
	/Users/runner/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:1044 +0x35c
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/runner/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:968
main.run()
	/Users/runner/work/bearer/bearer/cmd/bearer/main.go:21 +0x40
main.main()
	/Users/runner/work/bearer/bearer/cmd/bearer/main.go:13 +0x1c
```

## Possible Fix
This is just on main not present in 1.1 so probably related to recent changes presumably because the scan path is not passed in.

## Your Environment

<!--- Include as many relevant details about the environment you experienced the bug in -->

- Operating System and version:
- Output of 'bearer version':
```
bearer version: 0.00.0
sha: 089310d147862a14237345fc20e8eaba0a608e9f
```

--- a/pkg/commands/scan.go
+++ b/pkg/commands/scan.go
@@ -55,7 +55,10 @@ func NewScanCommand() *cobra.Command {
 			}
 
 			configPath := viper.GetString(flag.ConfigFileFlag.ConfigName)
-			defaultConfigPath := file.GetFullFilename(args[0], configPath)
+			var defaultConfigPath = ""
+			if len(args) == 1 {
+				defaultConfigPath = file.GetFullFilename(args[0], configPath)
+			}
 
 			var loadFileMessage string
 			if err := readConfig(configPath); err != nil {
@@ -98,6 +101,10 @@ func NewScanCommand() *cobra.Command {
 }
 
 func readConfig(configFile string) error {
+	if configFile == "" {
+		return nil
+	}
+
 	viper.SetConfigType("yaml")
 	viper.SetConfigFile(configFile)
 	if err := viper.ReadInConfig(); err != nil {

